src/coreclr/src/debug/createdump/CMakeLists.txt

```diff
@@ -2,53 +2,81 @@ project(createdump)
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
-include(configure.cmake)
-
-# Set the RPATH of createdump so that it can find dependencies without needing to set LD_LIBRARY_PATH
-# For more information: http://www.cmake.org/Wiki/CMake_RPATH_handling.
-if (CORECLR_SET_RPATH)
-  set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
-  if(CLR_CMAKE_HOST_OSX)
-    set(CMAKE_INSTALL_RPATH "@loader_path")
-  else()
-    set(CMAKE_INSTALL_RPATH "\$ORIGIN")
-  endif(CLR_CMAKE_HOST_OSX)
-endif (CORECLR_SET_RPATH)
+include_directories(BEFORE ${VM_DIR})
 
 remove_definitions(-DUNICODE)
 remove_definitions(-D_UNICODE)
 
-include_directories(BEFORE ${VM_DIR})
+if(CLR_CMAKE_HOST_WIN32)
+
+    # Use static crt
```

Comment:
I do not think we want static crt. And I do not think that this actually gives you static crt.

Looks like another copy&paste from dbgshim that does not make sense.

Reply:
Other exe/tools like GenClrDebugResource, InjectResource, superpmi all use static crt by both adding the -MT define and linking with ${STATIC_MT_CRT_LIB} and ${STATIC_MT_CPP_LIB}.

Is it the size of the resulting createdump.exe the problem? I seem to remember some obscure on some version of Windows that this solved that the right version of the CRT dlls weren't there.

Reply:
> GenClrDebugResource, InjectResource, superpmi

We do not ship these, so we do not care that they are bloated as much. And I do not think it that what the comment say anyway. These binaries depend on the dynamic CRT:

![image](https://user-images.githubusercontent.com/6668460/80250568-f5ddf780-8628-11ea-9a34-7494db60ce38.png)

> Is it the size of the resulting createdump.exe the problem?

We do not want to be adding to the shared framework needlessly. We do care about the size of the shared framework.